packages/genesys-spark-components/src/components/stable/gux-time-picker/tests/gux-time-picker.service.spec.ts

```diff
@@ -323,9 +323,9 @@ describe('gux-time-picker.service', () => {
         expectedOutput: GuxISOHourMinute[];
       }) => {
         it(`should work as expected for ${clockType} with ${minuteInterval} minute intervals`, async () => {
-          expect(getTimeDisplayValues(minuteInterval, clockType)).toStrictEqual(
-            expectedOutput
-          );
+          expect(
+            getTimeDisplayValues(minuteInterval, clockType, '0', '5:00')
```

Comment:
I'm going to write unit tests here and will not hardcode "0" and "5:00" after that